Webex Connect Flows/v3.0/Template/Event Handling Workflows/README.md

```diff
@@ -0,0 +1,155 @@
+As webex.variables and webex.RequestBody are nested variables , we are assigning these to response and requestBody respectively so that we can use them further.
+
+<img width="800" alt="WxCC Task v2 Transition Actions" src="../../images/WxCC Task v2 Transition Actions.png">
+
+
+##### Branch node
+
+<img width="800" alt="Branch" src="../../images/Branch.png">
+
+In the branch we are specifying if "webex.context" is add or remove that means if we are trying to add or remove participant from the
+contact then we can proceed further with the flow. In this sample we are showing screen pop if the "webex.context" is add or remove.
+
+### Closed Event flow
+
+* This workflow will be triggered when an agent or system has ended a contact and the conversation is closed.The event for this workflow is 'Task Closed'. So once the contact has been closed
+then this workflow will be triggered.
+
+#### When do you need Closed Event Flow ?
+* Closed event flow is an optional flow. It is needed only when you want to do some customization like Screen Pop or http calls when the contact has been ended.
+
+#### Start Node of Closed Event Flow
+
+<img width="800" alt="ClosedTaskv2" src="../../images/TaskClosed.png">
+
+Under the configuration section , we can see the event selected is Task Closed so this flow will be triggered on the Task Closed event. We can use the available variables from the
+output variables section.
+
+If you want to run the Closed Event flow to run only for specific asset then we need to enable the conditions checkbox and add condition to the "WxCC Task V2" Node.
+For enabling conditions please refer to the [Adding Conditions In Flow](#adding-conditions-in-flow) section.
+
+There is no restriction on the number of Closed Event Flow. You can add as many Closed Event Flow you need if different handling is required for each channel or asset.
+But in this case you need to specify condition for each flow.
+
+As webex.variables and webex.RequestBody are nested variables , we are assigning these to response and requestBody respectively so that we can use them further.
+
+<img width="800" alt="WxCC Task v2 Transition Actions" src="../../images/WxCC Task v2 Transition Actions.png">
+
+##### These event flows are applicable for all channels by default. If v2.x shared flows are already available in the same org, refer to the [Migration Strategy](#migration-strategy) section .
+
+## Adding Conditions In Flow
+If you want to run your flow to be triggered only if the specified conditions are met then it can be achieved by adding conditions.
+You can choose any condition or a set of conditions out of conditions shown in the dropdown of the below image based on your usecase.
+
+<img width="800" alt="ConditionsDropDown" src="../../images/ConditionsDropDown.png">
+
+### Examples of addition conditions in Flow.
+1. If you want to run a flow to run only for a particular channel let's say LiveChat then you can specify condition like webex.mediaChannel equals web.
+
```

Comment:
for a particular channel let's say LiveChat, then you can specify condition like "webex.mediaChannel" equals "web".